test/ssl-tests/27-ticket-appdata.conf.in

```diff
@@ -0,0 +1,57 @@
+# -*- mode: perl; -*-
+# Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+
+## Test session ticket app data
+
+use strict;
+use warnings;
+
+package ssltests;
+
+
+our @tests = (
+    {
+	"name" => "session-ticket-app-data",
+	"client" => {
+	    "MaxProtocol" => "TLSv1.3",
```

Comment:
Do we need to specify maxprotocol at all here? Or possibly we want to do both a TLSv1.3 test *and* a TLSv1.2 (assuming both are enabled) since the ticket mechanism is quite different between those protocol versions.

Reply:
I'll dup the tests.